tests/accidental_tests.ts

```diff
@@ -410,10 +412,10 @@ function cautionary(options: TestOptions): void {
 
   const accids = Object.values(accidentals).filter((accid) => accid !== '{' && accid !== '}');
   for (i = 0; i < staveCount; ++i) {
-    const stave = f.Stave({ x: 0, y: 10 + staveHeight / scale * i, width: staveWidth / scale });
+    const stave = f.Stave({ x: 0, y: 10 + (staveHeight / scale) * i, width: staveWidth / scale });
     const score = f.EasyScore();
     const rowMap = [];
-    for (j = 0; j < notesPerStave && (j + i * notesPerStave) < accids.length; ++j) {
+    for (j = 0; j < notesPerStave && j + i * notesPerStave < accids.length; ++j) {
```

Comment:
Can you explain these changes with parentheses?

The first change adds parentheses, but the second change removes parentheses. Is there anyway for us to make these lines more clear and future bug-proof?

Reply:
This is done automatically by eslint.